streams/src/test/java/org/apache/kafka/streams/integration/AdjustStreamThreadCountTest.java

```diff
@@ -61,15 +61,16 @@
     @Rule
     public TestName testName = new TestName();
 
+    private final List<KafkaStreams.State> stateToTransitions = new ArrayList<>();
```

Comment:
I think `stateHistory` or `stateTransitionHistory` would be a more meaningful name for this variable.

Reply:
Agree~